config/application.yml

```diff
@@ -0,0 +1 @@
+api_key: 'iXuv3TmRRccB1MSd0QHHDcRdE04vcF7FhXhEqh5t'
```

Comment:
careful not to check this in to Github! Figaro install should automatically add this to your .gitignore